bitbots_docs/docs/manual/tutorials/test_robot_hardware.rst

```diff
@@ -0,0 +1,44 @@
+================================================
+Testing the robot hardware and lowlevel software
+================================================
+
+Preliminaries
+-------------
+
+#. Use robot compile to flash correct version on robot
+#. Put robot in a safe spot, on a rope hanging from the ceiling
+#. Check if all cables are correctly connected
+#. Do the test in the provided order, to find out which part is faulty
+	
+Test hardware and ros_control
+-----------------------------
+#. Test IMU
+    roslaunch bitbots_ros_control ros_control_standalone.launch only_imu:=true
```

Comment:
```suggestion
    ``roslaunch bitbots_ros_control ros_control_standalone.launch only_imu:=true``
```

With the extra markdown, sphinx renders the commands as code or something similar